src/modules/core/producer_melt.c

```diff
@@ -418,6 +432,41 @@ mlt_producer producer_melt_init( mlt_profile profile, mlt_service_type type, con
 					mlt_properties_set_int( properties, "hide", 2 );
 			}
 		}
+		else if ( !strcmp( argv[ i ], "-chain" ) )
```

Comment:
The new `-chain` and `-link` options should be added to usage in `src/melt/melt.c` and `docs/melt.1`

Reply:
done